RxJavaTemplate/buildSrc/src/main/java/Versions.kt

```diff
@@ -0,0 +1,56 @@
+object Versions {
+     const val BUILD_GRADLE_VERSION = "4.2.2"
+
+     const val ANDROID_VERSION_CODE = 9
+     const val ANDROID_VERSION_NAME = "3.1.0"
+     const val ANDROID_MIN_SDK_VERSION = 23
+     const val ANDROID_TARGET_SDK_VERSION = 28
+     const val ANDROID_COMPILE_SDK_VERSION = 28
+
+     // Dependencies (Alphabet sorted)
+     const val ANDROID_CORE_VERSION = "1.2.0"
+     const val ANDROID_CRYPTO_VERSION = "1.0.0"
+     const val ANDROID_FRAGMENT_VERSION = "1.3.3"
+     const val ANDROID_LEGACY_SUPPORT_VERSION = "1.0.0"
+     const val ANDROID_LIFECYCLE_VERSION = "2.2.0"
+     const val ANDROID_MATERIAL_VERSION = "1.2.0-alpha06"
+     const val ANDROID_SUPPORT_VERSION = "1.2.0-beta01"
```

Comment:
Do we really need to use ANDROID_SUPPORT_VERSION ? As we used AndroidX already. Please consider about this, thank you.

Reply:
@doannimble 🤔 Sorry what do you mean by `As we used AndroidX already`? The current `ANDROID_SUPPORT_VERSION` is using for dependencies as below

```
            "androidx.appcompat:appcompat:${Versions.ANDROID_SUPPORT_VERSION}",
            "androidx.recyclerview:recyclerview:${Versions.ANDROID_SUPPORT_VERSION}",
```

I think that the `SUPPORT` makes you confused a bit. I would do rename for better meaning here 😄

Reply:
Yes please change to ANDROIDX_VERSION or something like that to avoid confusing.